blackbelt/deployment.py

```diff
@@ -9,11 +9,11 @@ def deploy_staging():
 
     post_message("Deploying branch %s to staging" % branch_name, "#deploy-queue")
 
-    check_call(['grunt', 'deploy', '--app=apiary-staging-pool1', '--force', "--branch=%s" % branch_name])
```

Comment:
@abtris I thought `-pool1` is now used for "testing", is that true or not?

Reply:
No, after rename `-pool1` will be plain staging.